modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/resources/com/liferay/headless/builder/resource/test/dependencies/expected_openapi.json

```diff
@@ -384,7 +384,7 @@
 			"url": "http://www.apache.org/licenses/LICENSE-2.0.html"
 		},
 		"title": "title",
-		"version": "v1.0"
```

Comment:
When did we stop sending the v1.0 as a version?

Reply:
I just realized something...